esmvaltool/recipes/recipe_combined_indices.yml

```diff
@@ -18,38 +19,26 @@ datasets:
   #- {dataset: bcc-csm1-1,  type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 2000,  end_year: 2002}
   #- {dataset: IPSL-CM5A-MR, type: exp, project: CMIP5,  mip: day,  exp: historical,  ensemble: r1i1p1,  start_year: 1981,  end_year: 2000}
 
-preprocessors:
-  preproc:
-    regrid:
-      target_grid: MPI-ESM-MR
-      scheme: linear
-    mask_fillvalues:
-      threshold_fraction: 0.95
-    extract_region:
-      end_latitude: 70
-      start_latitude: 20
-      end_longitude: 300
-      start_longitude: 200
-
+   
 diagnostics:
   combine_indices:
     description: Combine indices and/or compute area weighted averages.
     variables:
-      pr:
+      psl:
         preprocessor: preproc
```

Comment:
Also remove the line `preprocessor: preproc`, or the recipe will fail complaining that it cannot find preprocessor `preproc`